PSEUDO API 1.md

```diff
@@ -0,0 +1,166 @@
+
+## XR
+
+	interface XR {
+		Promise<sequence<XRDisplay>> getDisplays();
+	};
+
+
+## XRDisplay
+
+	interface XRDisplay {
+		readonly attribute DOMString displayName;
+		readonly attribute boolean isExternal;
+
+		Promise<boolean> supportsSession(XRSessionCreateParametersInit parameters);
+		Promise<XRSession> requestSession(XRSessionCreateParametersInit parameters);
+
+		attribute EventHandler ondeactivate;
+	};
+
+### Todo
+
+- calibration
+
+## Reality
+
+	interface Reality {
+		readonly atttribute DOMString realityName;
+		readonly attribute boolean hasPointCloud;
+		readonly attribute XRCoordinates stageLocation;
+
+		Promise<boolean> requestStageLocation(float x, float y float z, XRCoordinates)
+		Promise<boolean> requestResetStageLocation()
+
+		Promise<XRPointCloud> getPointCloud()
+
+		Promise<XRLightEstimate> getLightEstimate()
+
+		attribute EventHandler onchange;
+	};
+
+### Todo
+
+- configuration
+- anchors
+- manifold instead of point cloud
+
+## XRPointCloud
+
+	interface XRPointCloud {
+		readonly attribute Float32Array points;
+	}
+
+## XRLightEstimate
```

Comment:
Apple's ARKit exposes these values, which allow layers to render their objects with appropriate colors.
Perhaps we should also expose sun direction by using the time of day and Coordinates?